raml-maven-plugin-it/pom.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>uk.gov.justice.maven</groupId>
+    <artifactId>raml-maven-plugin-it</artifactId>
+    <version>1.1.1-SNAPSHOT</version>
+
+    <!-- TODO add parent after extracting framewrok specific parent pom that does not contain raml-maven-plugin configuration-->
+    <!-- remember to manually update version -->
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                </configuration>
+            </plugin>
```

Comment:
This pom has no parent because the raml plugin configured currently in the parrent interferes with the integration test.
I will add parent when we extract a framework specific parent pom and bom.